sdks/python/apache_beam/runners/portability/fn_api_runner/translations.py

```diff
@@ -562,9 +569,10 @@ def copy_subtransforms(transform):
       copy_subtransforms(transform)
     else:
       transform = stage.executable_stage_transform(
-          known_runner_urns, all_consumers, components)
+          known_runner_urns, all_consumers, pipeline_proto.components)
```

Comment:
Is it possible that `components` has additional components that `pipeline_proto.components` does not?

Reply:
Yes but this is still safe to do. The only additional components in `components` but not `pipeline_proto.components` are executable stage transforms, and these are not used within `executable_stage_transform()` - in fact `executable_stage_transform()` does not use the transforms in the passed in components at all.